src/Tools/Tools.jsx

```diff
@@ -235,19 +235,21 @@ class Tools extends Component {
                   isExpanded={true}
                 />
               </div>
-              {COMMERCIAL_DATA_ENABLED && userAccountInfo?.payingAccount && (
-                <Tab
-                  id="commercial-tab"
-                  title={t`Commercial`}
-                  enabled={userAccountInfo?.payingAccount}
-                  renderKey={TABS.COMMERCIAL_TAB}
-                >
-                  <div className="commercial-data-wrapper">
-                    <CommercialData userAccountInfo={userAccountInfo || {}} />
-                  </div>
-                </Tab>
-              )}
             </Tab>
+            {/*This check disabled for now. Waiting on role check response from API*/}
+            {/*{COMMERCIAL_DATA_ENABLED && userAccountInfo?.payingAccount && (*/}
+            <Tab
+              id="commercial-data-tab"
```

Comment:
`id` of the tab should be also changed just for the sake of consistency, same for `className` below

Reply:
I totally overlooked that. It is fixed now. Thanks